tck/README.md

```diff
@@ -0,0 +1,117 @@
+
+## Appeals process
+
+Any test case (i.e. `@Test` method), test case configuration (e.g. `@Deployment`), test entities, annotations and other resources may be challenged by an appeal.
+
+To initiate a challenge, refer to the [Jakarta EE TCK Process 1.0](https://jakarta.ee/committees/specification/tckprocess/)
+
+## Installation
+
+This chapter explains how to obtain the TCK and supporting software and provides recommendations for how to install/extract it on your system.
+
+### Obtaining the Software
+
+You can obtain a release of the Jakarta JSON-B TCK from Maven Central at the following coordinates:
+
+```xml
+<dependency>
+  <groupId>jakarta.json.bind</groupId>
+  <artifactId>jakarta.json.bind-tck</artifactId>
+  <version>${jsonb.version}</version>
+  <scope>test</scope>
+</dependency>
+```
+
+### The TCK Environment
+
+To successfully run the JSON-B TCK, the following components are required:
+
+- [Apache Maven](https://maven.apache.org/install.html)
+- Java 8 or newer
+- An implementation of JSON-B
+- An implementation of CDI
+
+The rest of the TCK software can simply be obtained via Maven dependency described in the "Obtaining the Software" section.
+
+## Configuration
+
+To configure the JSON-B TCK, the following elements are required:
+
+1. A Maven `<dependency>` that pulls in the `jakarta.json.bind-tck` as described in the "Obtaining the Software" section
+2. A Maven `<dependency>` that pulls in an implementation of JSON-B
+3. A Maven `<dependency>` that pulls in an implementation of CDI (may be from the same depenency as (2))
+4. A Maven `<dependency>` that pulls in an appropriate Arquillian container adapter
+5. Configuration of the `maven-surefire-plugin` to include running the JSON-B TCK tests:
+
+    ```
+    <plugin>
+      <artifactId>maven-surefire-plugin</artifactId>
+      <version>3.0.0-M4</version>
+      <configuration>
+        <dependenciesToScan>
+          <dependency>jakarta.json.bind:jakarta.json.bind-tck</dependency>
+        </dependenciesToScan>
+      </configuration>
+    </plugin>
+    ```
+    
+6. Any runtime-specific configuration needed in order to enable JSON-B functionality
+
+## Reports
+
+Since tests are run using the `maven-surefire-plugin`, test reports can also be produced and configured using the [`maven-surefire-report-plugin`](https://maven.apache.org/surefire/maven-surefire-report-plugin/)
+
+## Running the TCK tests
+
+Once the TCK has been configured for your runtime and all required software has been installed, the TCKs can be executed by running:
+
+```
+$ mvn test
+```
+
+All test execution follows standard Maven Surefire practices, such as:
+ - Run a single test class with `mvn test -Dtest=MyTestClass
+ - Run a singl etest method with `mvn test -Dtest-MyTestClass#myTestMethod
+ 
+## Example setup
+
+See the [Eclipse Yasson repository](https://github.com/eclipse-ee4j/yasson/tree/master/yasson-tck) for a working example of how a JSON-B implementation can consume and run these TCK tests.
```

Comment:
This link is currently broken, but will be active once I merge the corresponding PR in Yasson